vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -335,7 +335,67 @@ class Button extends React.Component<
     //  Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
     // Windows]
     // [Windows
-    if (Platform.OS === 'windows') {
+    if (Platform.OS === 'windows' && color) {
+      return (
+        <Touchable
+          accessibilityLabel={accessibilityLabel}
+          accessibilityRole="button"
+          accessibilityState={accessibilityState}
+          hasTVPreferredFocus={hasTVPreferredFocus}
+          nextFocusDown={nextFocusDown}
+          nextFocusForward={nextFocusForward}
+          nextFocusLeft={nextFocusLeft}
+          nextFocusRight={nextFocusRight}
+          nextFocusUp={nextFocusUp}
+          testID={testID}
+          disabled={disabled}
+          onPress={onPress}
+          tabIndex={tabIndex}
+          touchSoundDisabled={touchSoundDisabled}
+          underlayColor="white"
```

Comment:
this doesn't seem right always (e.g. dark mode). I see the other case uses `underlayColor={PlatformColor('ButtonBackgroundPressed')}`

Reply:
When the button is pressed, the TouchableHighlight component take its contents at rest and adds opacity to the contents making the underlayColor more visible. In this case, it will appear as if the colored button has lightened slightly. ButtonBackgroundPressed is a very translucent grey which in this scenario won't give us the effect we want. Since for the colored button case the background will always be the input color, leaving this as white will not cause large visual breaks, but let me look into finding a brush that grabs application background. This would allow us to more closely match the XAML Button case for an accent button.